lib/sshkit/scp/command.ex

```diff
@@ -3,22 +3,29 @@ defmodule SSHKit.SCP.Command do
 
   import SSHKit.Utils
 
-  def build(mode, path, options) do
-    scp = case mode do
-      :upload -> 'scp -t'
-      :download -> 'scp -f'
-    end
+  @flags [verbose: '-v', preserve: '-p', recursive: '-r']
 
-    flags = [verbose: '-v', preserve: '-p', recursive: '-r']
+  def build(:upload, path, options) do
+    scp('-t', path, options)
```

Comment:
doesn't really improve things. I'm more for a link to the most official documentation of these undocumented flags in comments :) so people know where to look for an explanation and further options. :) thanks for the links btw